compiler/x/codegen/BinaryEvaluator.cpp

```diff
@@ -1498,7 +1498,7 @@ TR::Register *OMR::X86::TreeEvaluator::bsubEvaluator(TR::Node *node, TR::CodeGen
 
    if (NEED_CC(node))
       {
-      TR_ASSERT(node->getOpCodeValue() == TR::bsub,
+      TR_ASSERT(node->getOpCodeValue() == TR::bsub | TR::busub,
```

Comment:
I don't think this assert is right: should be TR_ASSERT(node->getOpCodeValue() == TR::bsub || node->getOpCodeValue() == TR::bsub, ...